### PR TITLE
FAQ - where can I get tested: responsive images and two columns

### PR DESCRIPTION
**Pull request by dsarkar**
Internal Tracking ID: [EXPOSUREAPP-9022](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9022)

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -65,14 +65,11 @@
...
+                            "<hr/>",
+                            "<div class='container'>  <div class='row'>    <div class='col-sm'><a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/BMG_testzentren.png' alt='image'></a></div><div class='col-sm'>An overview of the <b>PCR and rapid antigen test sites</b> can be found on this <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG webpage</a> (not clear whether the test site is connected to the CWA or not).</div></div>",
+                            "<hr/>",
+                            "Another list of test sites from the GitHub community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"                        ]
```
Comment (Ein-Tim) [inline src/data/faq.json:72]:
```suggestion
                            "Another list of test sites from the GitHub community: <a href='https://github.com/corona-warn-app/cwa-documentation/issues/569' target='_blank' rel='noopener noreferrer'>Where to get tested?</a>"                        
]
```

Move the bracket down (:

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

**Comment by dsarkar** [comment]
Looks good now on small screens: iPhone 6s example

Landscape:

image

Portrait:

image

**Comment by dsarkar** [comment]
@Ein-Tim You finished with the review ?

**Comment by Ein-Tim** [comment]
@dsarkar Yes 👍

**Review by Ein-Tim — APPROVED** [review]

**Comment by Ein-Tim** [comment]
LGTM!

**Pull request merged**